tool/README.MD

```diff
@@ -0,0 +1,10 @@
+ ## Generate the json configuration file used by GNI tool
+ 
+ * [util.py](./util.py): functions used for building the property-value tree
+ * [stat_config.proto](./stat_config.proto): configuration file for generating [stat_config_pb2.py](./stat_config_pb2.py), used for parsing the input config files. 
+ * [run_local.sh](./run_local.sh): commands for running the script locally
+ 
+ * [pop_obs_spec_common.textproto](./pop_obs_spec_common.textproto):  input config file
```

Comment:
what is the proto spec for these files? could you describe more about the inputs, esp what is expected.